pkg/utils/kubernetes/health/etcd.go

```diff
@@ -22,10 +22,24 @@ import (
 )
 
 // CheckEtcd checks whether the given Etcd is healthy.
-// A Etcd is considered healthy if its ready field in status is true.
+// An Etcd is considered healthy if its ready field in status is true and the BackupReady condition doesn't report false.
 func CheckEtcd(etcd *druidv1alpha1.Etcd) error {
 	if !pointer.BoolDeref(etcd.Status.Ready, false) {
 		return fmt.Errorf("etcd %q is not ready yet", etcd.Name)
 	}
+
+	for _, cond := range etcd.Status.Conditions {
+		if cond.Type != druidv1alpha1.ConditionTypeBackupReady {
+			continue
+		}
+		if cond.Status == druidv1alpha1.ConditionFalse {
```

Comment:
What about `Unknown`? WDYT about
```suggestion
		if cond.Status != druidv1alpha1.ConditionTrue {
```
Even if this cannot happen today, maybe tomorrow it can and then we are prepared :)

Reply:
This is what I planned to do at the beginning. However, I noticed an issue with `etcd-events` where the condition remains at status `Unkown` and thus decided to just check for `ConditionFalse` for now.
I opened https://github.com/gardener/etcd-druid/issues/413 and added a TODO to change this handling as soon as the issue is revolved. Hope that's fine 🙂